### ISSUE TITLE
SHOW output should be alphabetically sorted

### ISSUE THREAD

**Issue by rmoff** [confluentinc/ksql#2973]
The output of `SHOW STREAMS` / `SHOW TABLES` etc is currently unsorted making it difficult to use especially as the number of objects increases. 

```
ksql> show streams;

 Stream Name                             | Kafka Topic                             | Format
--------------------------------------------------------------------------------------------
 TRAIN_ACTIVATIONS_00                    | TRAIN_ACTIVATIONS_00                    | AVRO
 TRAIN_MOVEMENTS_ACTIVATIONS_00          | TRAIN_MOVEMENTS_ACTIVATIONS_00          | AVRO
 TRAIN_CANCELLATIONS_02                  | TRAIN_CANCELLATIONS_02                  | AVRO
 CORPUS_RAW                              | corpus                                  | JSON
 TRAIN_ACTIVATIONS_01                    | TRAIN_ACTIVATIONS_01                    | AVRO
 CORPUS_BY_STANOX                        | CORPUS_BY_STANOX                        | AVRO
 TRAIN_CANCELLATIONS_01                  | TRAIN_CANCELLATIONS_01                  | AVRO
 TRAIN_MOVEMENTS_01                      | TRAIN_MOVEMENTS_01                      | AVRO
 TRAIN_CANCELLATIONS_00                  | TRAIN_CANCELLATIONS_00                  | AVRO
 TRAIN_MOVEMENTS_00                      | TRAIN_MOVEMENTS_00                      | AVRO
 NETWORKRAIL_TRAIN_MVT_X                 | networkrail_TRAIN_MVT_X                 | JSON
 TIPLOC_FLAT_KEYED                       | TIPLOC_FLAT_KEYED                       | AVRO
 SCHEDULE_RAW                            | CIF_FULL_DAILY                          | JSON
 TRAIN_MOVEMENTS_ACTIVATIONS_SCHEDULE_03 | TRAIN_MOVEMENTS_ACTIVATIONS_SCHEDULE_03 | AVRO
 TRAIN_MOVEMENTS_ACTIVATIONS_SCHEDULE_01 | TRAIN_MOVEMENTS_ACTIVATIONS_SCHEDULE_01 | AVRO
 SCHEDULE_00                             | SCHEDULE_00                             | AVRO
 TRAINS_DELAYED_OVER_30MIN               | TRAINS_DELAYED_OVER_30MIN               | AVRO
 TRAIN_MOVEMENTS_ACTIVATIONS_SCHEDULE_00 | TRAIN_MOVEMENTS_ACTIVATIONS_SCHEDULE_00 | AVRO
 SCHEDULE_01                             | SCHEDULE_01                             | AVRO
```

We should make this list alphabetically sorted instead.
